packages/generators/templates/component/{{name}}.{{mdx}}

```diff
@@ -28,6 +28,16 @@ import { {{ name }} } from "@jobber/components/{{ name }}";
 
 ---
 
+## Design Guidelines
+
+When contributing to the {{ name }} component, follow these guidelines:
+
```

Comment:
I like the sections here. I wonder if we will get more quality information added (for both design guidelines and best practices) if we provide some examples, or even link to a component file/doc that does a good job of filling these in.

Reply:
I wonder if @chris-at-jobber would have any thoughts along those lines. Even if the example was types of things to consider or something?

Reply:
To me this is a nice addition, it seems to be mirroring the placement from the [Icons PR](https://github.com/GetJobber/atlantis/pull/219) but outside of that context I think "design guidelines" is equivalent to usage guidelines a la [Button](https://atlantis.frend.space/components/button#work-actions). 

Icons are interesting in that they're such a primarily visual component, the structural notes there are  more structural guidelines than usage guidelines for the designers. 

---

# EXAMPLE

_Label | Label | Label_
[ import snippet thing ]

A paragraph section for the component description

[ Component Sandbox chunk ]

## Design Guidelines

This section would include things like Work Actions, Learning Actions, etc. 

## Props

Props stuff

---

I realize that this makes it a more design-centric way of documenting, but I feel like putting usage guidelines before technical guides could help teams better identify if a given component is right for their use case.

Reply:
Do you see the BEST PRACTICES section serving the role I'm talking about here?

Reply:
In regards to best practices, I really like what shopify does on their [components](https://polaris.shopify.com/components/forms/autocomplete#section-best-practices). They give information about how to use and *not use* each component.

Do you all think this would be a good place to keep items like this? In my mind "best practices" would continue to grow per component as consumers have questions about the component.